### PR TITLE
To do List (List structure)

### PR DESCRIPTION
**Pull request by umair-rao**
In this pull request, I have built a to-do list according to the following instructions.

 - Created an index.html file and wrote HTML markup using the HTML Webpack Plugin.
 - Create an index.js file and added a function to create a list of tasks for the To-do list.
 
 Please check PR.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
-	return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+window.addEventListener('load', () => {
+  const form = document.querySelector('.add-list');
+  const input = document.querySelector('#item-add');
+  const listEl = document.querySelector('.taskContainer');
+
```
Comment (franclobo) [inline src/index.js]: - [ ] Kindly create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object should contain three keys, to complete the list dynamically:

- description [string].
- completed [bool].
- index: [number].

File: src/index.js [hunk 2]
```diff
@@ -1,20 +1,35 @@
...
+    const task = input.value;
+
+    const taskEl = document.createElement('div');
+    taskEl.classList.add('task');
+
+    const taskContent = document.createElement('div');
+    taskContent.classList.add('list-items');
+
+    taskEl.appendChild(taskContent);
```
Comment (franclobo) [inline src/index.js]: - [ ] Please 🙏🏽 write a function to iterate over the tasks array and populate an HTML list item element for each task, it is a project requirement.

File: src/index.js [hunk 3]
```diff
@@ -1,20 +1,35 @@
...
+
+    const taskInput = document.createElement('input');
+    taskInput.classList.add('add-item');
+    taskContent.type = 'text';
+    taskInput.value = task;
+    taskInput.setAttribute('readonly', 'readonly');
+
+    taskContent.appendChild(taskInput);
+
+    listEl.appendChild(taskContent);
```
Comment (franclobo) [inline src/index.js]: - [ ] Please 🙏🏽 complete the project requirements to approve it.

> On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @umair-rao,

# Required Changes ♻️ 

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✅ 
- Descriptive PR ☑️ 
- Professional README file 💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Please 🙏🏽 complete the functions requested for the `index.js` file to add the task dynamically.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arslanbisharat** [comment]
Hi @umair-rao,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**